libs/utils/src/lib/spreadsheet/import.ts

```diff
@@ -241,7 +242,28 @@ export function getStaticList(scope: Scope, value: string, separator:string, nam
   return values;
 }
 
-export function split(cell: string, separator:string) {
+export function getGroupedList<GS extends GroupScope>(value: string, groupScope: GS, separator: string) {
+  const elements = split(value, separator);
+  const groupLabels = staticGroups[groupScope].map(group => group.label);
+  let allElements = elements.map(element => {
+    if (groupLabels.includes(element)) {
+      const groups = staticGroups[groupScope] as StaticGroup<GS>[]
+      return groups.find(group => group.label === element).items;
+    }
+    return [element];
```

Comment:
as `groupScope` is typed you should be able to do : 
```suggestion
    return groupLabels.includes(element)
      ? staticGroups[groupScope].find(group => group.label === element).items
      : element;
```